src/test/java/org/mule/extension/sftp/PostActionGroupTestCase.java

```diff
@@ -40,4 +63,194 @@ public void deleteAndMove() {
   public void deleteAndRename() {
     new PostActionGroup(true, null, "thisone.txt", true).validateSelf();
   }
+
+
+  @Test(expected = FileAlreadyExistsException.class)
+  @Description("verifies that move file with overwrite flag set to false and exceptiopn will be thrown")
+  public void testMoveWithoutOverwriteWhenFileExists() {
+    FileAttributes fileAttributes = mock(FileAttributes.class);
+    when(fileAttributes.getName()).thenReturn("");
+    FileConnectorConfig fileConnectorConfig = mock(FileConnectorConfig.class);
+
+    ConcreteFileSystem fileSystem = new ConcreteFileSystem("");
+
+    new PostActionGroupChild("someDir", null, false, false).apply(fileSystem, fileAttributes, fileConnectorConfig);
+  }
+
+  @Test
+  @Description("verifies that move file with overwrite flag set to true and no exception")
+  public void testMoveWithOverwriteWhenFileExists() {
+    FileAttributes fileAttributes = mock(FileAttributes.class);
+    when(fileAttributes.getName()).thenReturn("");
+    FileConnectorConfig fileConnectorConfig = mock(FileConnectorConfig.class);
+
+    ConcreteFileSystem fileSystem = new ConcreteFileSystem("");
+    fileSystem.setCanMove(true);
+    new PostActionGroupChild("someDir", null, false, true).apply(fileSystem, fileAttributes, fileConnectorConfig);
```

Comment:
You should add at least an assertion here

Reply:
What about the PR yesterday.... I wonder if I saw this test and I wrote the comment in the other one...

Reply:
OK, sorry!!!
Got it, just updated.

Reply:
I see. I just used the SonarLint and found the assertion error.
But I don't know what kind of assertion I should put here, so I search online and found that the "expected" is what I need to use.
And then the SonarLint error is resolved now.